plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/model/CodeModernizerArtifact.kt

```diff
@@ -53,36 +55,37 @@ open class CodeModernizerArtifact(
                     throw RuntimeException("Could not unzip artifact")
                 }
                 val manifest = loadManifest()
-                if (manifest.version > maxSupportedVersion) {
+                if (manifest.version > MAX_SUPPORTED_VERSION) {
                     // If not supported we can still try to use it, i.e. the versions should largely be backwards compatible
                     LOG.warn { "Unsupported version: ${manifest.version}" }
                 }
                 val patches = extractPatches(manifest)
                 val summary = extractSummary(manifest)
                 val summaryMarkdownFile = getSummaryFile(manifest)
+                val metrics = loadMetrics(manifest)
```

Comment:
Make sure backend is returning a `metrics.json` for SQL conversions, otherwise will have to wrap this in a `try-catch`